src/DynamoCoreWpf/ViewModels/Core/StateMachine.cs

```diff
@@ -26,7 +26,8 @@ partial class WorkspaceViewModel
         #region State Machine Related Methods/Data Members
 
         private StateMachine stateMachine = null;
-        private ConnectorViewModel activeConnector = null;
+        private ConnectorViewModel[] activeConnectors = null;
```

Comment:
Please put some comments about scenarios where there will be one activeConnector and when it will have multiple active connectors.